### PR TITLE
[thephoenixbay.com] Update and reactivate ruleset.

### PR DESCRIPTION
**Pull request by bardiharborow**
1. Pulled the latest list from the main site.
2. Pruned all targets that failed the fetch test.
3. Added a `www` target for each domain.
4. Pruned all `www` targets that failed the fetch test.
5. Prune duplicate and preloaded targets.

### CODE REVIEW

**Comment by J0WI** [comment]
#11528 tries to split them into different files. As you can see there, not all of them have exact the same subdomains/behavior.

**Comment by bardiharborow** [comment]
@J0WI This mainly touches `thephoenixbay.com.xml`, so I've just reworked the edits to remove duplicated targets from `thephoenixbay.com.xml` rather than touching `proxybay.one.xml`. We may want to rewrite these as individual rulesets later, but this will at least get a chunk of domains secured again.

File: src/chrome/content/rules/thephoenixbay.com.xml [hunk 1]
```diff
@@ -1,623 +1,357 @@
...
+        - unblockbay.co (no https)
+        - unblockbay.link (tls error)
+        - unblockbay.pw (mismatch)
+        - unblockbay.xyz (mismatch)
+        - unblockthepiratebay.net (timeout)
+        - unblockthepiratebay.org (timeout)
+        - unblocktpb.co (no https)
+        - unblocktpb.link (dns dead)
+        - unblocktpb.pw (expired)
+        - unblocktpb.xyz (self-signed)
```
Comment (J0WI) [inline src/chrome/content/rules/thephoenixbay.com.xml]: You can remove all "dns dead" hosts from the list.

**Comment by koops76** [comment]
This should be split into individual files for each domain prior to merging. I can do it automatically and make a PR to your fork with all of them split out.

**Comment by bardiharborow** [comment]
@koops76 I guess I saw that as a lot of effort and maintenance burden for what are mostly just rapidly outdated throwaway domains, but if that's the way you'd prefer to do it feel free to shoot me a PR. :)

**Comment by koops76** [comment]
@bardiharborow See bardiharborow#1.

**Comment by koops76** [comment]
@bardiharborow See Travis.

**Comment by bardiharborow** [comment]
Travis is currently experiencing data center issues, and that may be contributing to the false positive timeouts.

**Comment by bardiharborow** [comment]
@J0WI This should be mergeable now. I've had to remove 5 rulesets which pass locally, but are consistently failing Travis. I'll put in a separate PR in a moment.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**